comp/core/workloadmeta/collectors/internal/kubeapiserver/kubeapiserver_test.go

```diff
@@ -373,7 +402,7 @@ func TestResourcesWithMetadataCollectionEnabled(t *testing.T) {
 				"language_detection.enabled":                       true,
 				"language_detection.reporting.enabled":             true,
 				"cluster_agent.kube_metadata_collection.enabled":   true,
-				"cluster_agent.kube_metadata_collection.resources": "daemonsets deployments",
+				"cluster_agent.kube_metadata_collection.resources": "apps/daemonsets apps/deployments",
```

Comment:
I think the `expectedResources` attribute in these tests needs to be adapted to the new format.

Reply:
thanks 👍 

updated